src/js/components/locations-configuration/LocationDetails.jsx

```diff
@@ -210,6 +214,55 @@ class LocationDetails extends Component {
       .value();
   }
 
+  fetchLocation() {
+    const url = `/openboxes/api/locations/${this.props.match.params.locationId}`;
+    apiClient.get(url).then((response) => {
+      const location = response.data.data;
+      if (!location) {
+        this.props.history.push('/openboxes/locationsConfiguration/create');
+        return;
+      }
+      const splitLocationTypeName = (locationType) => {
```

Comment:
@kchelstowski move this to utils please, so it could be used in other places. It could take a text parameter and a locale parameter. It always felt weird that we had to do this. 

@jmiranda why some names in the system are using these weird type of i18n? Could not it be in the description field when it's needed? Or perhaps there is a proper approach to such translations that we could implement.